shared/util/quit-helper.desktop.js

```diff
@@ -10,7 +11,7 @@ export type Action = 'closePopups' | 'quitMainWindow' | 'quitApp'
 function quitOnContext(context: Context): Array<Action> {
   switch (context) {
     case 'uiWindow':
-      return ['closePopups', 'quitMainWindow']
+      return isWindows ? ['quitApp'] : ['closePopups', 'quitMainWindow']
```

Comment:
I support this change, although I've asked for it in the past and experienced pushback.

Reply:
what is motivating this change

Reply:
These 2 menu items seem to do the same thing, and people are cranky that "quit' doesn't quit the app. Could also be solved by just removing "quit".
![image](https://user-images.githubusercontent.com/862397/52737350-043e7700-2f81-11e9-98e7-fd980b741013.png)

Reply:
this is how it works on all the other platforms and is a behavior we decided on in design a long time ago. if you want to convince them we should change go for it but lets not change the UX on window just cause